### PR TITLE
front: manage invalid trains

### LINKED ISSUE #4551: front: invalid a train if infra or rolling_stock invalid
# Description

When an infra or a rolling_stock has an invalid version, the trains that are linked to this infra or rolling_stock should be marked as invalid with hatching.
The user should know that trains are invalid and that they have to be removed.

To get an invalid train, add a train with a custom rollingstock and then modify his rollingstock in the rollingstock editor (not the name) or modify something in the infra used in the scenario.

[Avoir une gestion des trains invalides](https://github.com/orgs/osrd-project/projects/12/views/6?filterQuery=-status%3A%22%E2%9C%85+Done%22+gestion&pane=issue&itemId=31875549)

# Acceptance Criteria

- [ ] invalid trains should be displayed with hatching in the timetable (train list)
- [ ] When hovering an invalid train, the remove button should be the only one visible
- [ ] A message "La grille horaires comporte des trains invalides." should be displayed at the bottom left of the timetable, above the conflict list if there is at least one invalid train in the least
- [ ] Pictogramms (one for newer infra and one for newer rolling stock) should be displayed for each train for each reason of the invalidation)

### PR DESCRIPTION
**Pull request by SharglutDev**
close #4551

### CODE REVIEW

File: front/public/locales/fr/operationalStudies/scenario.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "delete": "Supprimer",
     "duplicate": "Dupliquer",
     "importTrainSchedule": "Importer",
+    "invalidTrains": "La grille horaires comporte des trains invalides.",
```
Comment (clarani) [inline front/public/locales/fr/operationalStudies/scenario.json:38]: You forgot the English translation :)

**Review by clarani — COMMENTED** [review]
LGTM, tested and everything works ! Nice job :muscle: I only left 1 comment

Do you think you can change the text in the pophover ? :)
image

**Review by Akctarus — APPROVED** [review]
tested and LGTM, good job !

File: front/public/locales/fr/operationalStudies/scenario.json (hunk 1 above)
Comment (SharglutDev) [reply front/public/locales/fr/operationalStudies/scenario.json:38]: Done :)

Comment (SharglutDev) [reply front/public/locales/fr/operationalStudies/scenario.json:38]: For the text in the popover, it has been decided during refinement and talks with operational studies to use the back error messages for now

**Review by clarani — COMMENTED** [review]
LGTM !!

**Review by clarani — APPROVED** [review]

**Pull request merged**